churner/churner.go

```diff
@@ -178,9 +178,8 @@ func (c *churner) getOperatorsToChurn(ctx context.Context, quorumIDs []uint8, op
 			return nil, errors.New("maxOperatorCount is 0")
 		}
 
-		if uint32(len(operatorStakes[quorumID])) < operatorSetParams.MaxOperatorCount {
-			// quorum is not full, so we can continue
-			continue
+		if len(operatorStakes[quorumID]) == 0 {
```

Comment:
1. So this would cause registration to fail if there was one empty quorum and another quorum which was full. Can't we handle this case by just passing in an empty operators to churn object? 

2. Shouldn't we also pass in an empty object whenever the quorum isn't full? It doesn't look like the contracts require the object to be populated in that case, but maybe there's an issue I'm not seeing.

Reply:
I like that idea.
@0x0aa0 can you chime in here? 
Can we have zero address for operator address in [operatorKickParams](https://github.com/Layr-Labs/eigenlayer-middleware/blob/dev/src/RegistryCoordinator.sol#L181) in case the quorum isn't full?

Reply:
Done

Reply:
We check that the operator passed in the kickParams is registered so it would fail [here](https://github.com/Layr-Labs/eigenlayer-middleware/blob/dev/src/RegistryCoordinator.sol#L568).

Reply:
@0x0aa0 , but in these situations, the call to `deregisterOperator` couldn't be occurring...